sdk/nodejs/cmd/pulumi-language-nodejs/testdata/tsc/sdks/plain-13.0.0/types/input.ts

```diff
@@ -0,0 +1,25 @@
+// *** WARNING: this file was generated by pulumi-language-nodejs. ***
+// *** Do not edit by hand unless you're certain you know what you are doing! ***
+
+import * as pulumi from "@pulumi/pulumi";
+import * as inputs from "../types/input";
+import * as outputs from "../types/output";
+
+export interface DataArgs {
+    boolArray: boolean[];
```

Comment:
Do we have any conformance tests where the container is plain but the values are inputs?
- `Input<boolean>[]`
- `{[key]: string]: Input<string>}`

It's common to want to do that for packaged components (MLCs) so you can loop over the container to create child resources without needing to do it inside an `apply`.

Reply:
No but we could add that, would probably make sense to just extend this test?